homeassistant/components/transmission/config_flow.py

```diff
@@ -59,18 +59,14 @@ async def async_step_user(self, user_input=None):
                 if (
                     entry.data[CONF_HOST] == user_input[CONF_HOST]
                     and entry.data[CONF_PORT] == user_input[CONF_PORT]
-                ):
-                    return self.async_abort(reason="already_configured")
-                if entry.data[CONF_NAME] == user_input[CONF_NAME]:
-                    errors[CONF_NAME] = "name_exists"
-                    break
-
```

Comment:
Why is that removed ?

Reply:
Not removed as noted in PR info it was unified. All aborts happens due to entry already exists leading to "single_instance_allowed" abort message. Saw similar unification in other PR for config flow to avoid having too many specialized error messages telling user the same info. Ofc can undo if it is too much.

Reply:
👍

Reply:
I think single_instance_allowed would not be the right error here right? It seems multiple instances are allowed as long as they have unique host/port and name. already_configured would be more appropriate as abort reason.

Reply:
@RobBie1221 good point. I didn't think about single_instance like it does not allow more host/port configurations. Now it make sense and I changed the reason to already_configured_device. Thanks.

Reply:
@Pigotka I favor separate errors for name and host/port.  On the occasion of the name_exists error it is likely that the user is configuring multiple Transmission instances and forgot to give the new instance a new name.  On the occasion of already_configured or already_configured_device error it is likely that the user had already configured Transmission and forgot what they named it.

Reply:
An existing name used to trigger an error, now it aborts the flow. It should be reverted to just triggering an error again.

Reply:
Done.